Makefile

```diff
@@ -307,3 +345,72 @@ verify-bundle: bundle ## Verify bundle update.
 .PHONY: verify-fmt
 verify-fmt: fmt ## Verify fmt update.
 	git diff --exit-code ./api ./controllers
+
+## local configurations
+
+deploy-develmode: manifests kustomize ## Deploy controller in debug mode to the K8s cluster specified in ~/.kube/config.
```

Comment:
The target name says "devel" mode while the descriptions says "debug". But, most importantly:
- What's the difference to the existing `deploy` target?
- Why does it override the deployment manifests file at `$(DEPLOYMENT_FILE)`, but then won't use it?

Reply:
At the I am going to say I had a valid reason for doing it but I can not find it now. I will remove and use the `deploy` target.